src/goto-symex/path_storage.cpp

```diff
@@ -29,12 +29,9 @@ path_storaget::patht &path_lifot::private_peek()
   return paths.back();
 }
 
-void path_lifot::push(
-  const path_storaget::patht &next_instruction,
-  const path_storaget::patht &jump_target)
+void path_lifot::push(const path_storaget::patht &path)
```

Comment:
I don't think we actually need these overrides anymore.

Reply:
The interface doesn't define the storage used.